testdata/testFiles/hello2.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this worth the hassle vs just making a dummy file using a TempDirectory?

Maybe it is worth it for project "templates" for future tests...

Reply:
There's not hassle here though? I'd like to redo how we do this later but not in this PR